src/amo/components/UserProfile/index.js

```diff
@@ -307,10 +431,14 @@ export function mapStateToProps(
     user &&
     (currentUser.id === user.id || hasPermission(state, USERS_EDIT));
 
+  const reviews = user ? getReviewsByUserId(state.reviews, user.id) : null;
+
   return {
-    currentUser,
     canEditProfile,
+    currentUser,
     isOwner,
+    reviewCount: reviews ? reviews.reviewCount : null,
+    reviews: reviews ? reviews.reviews.slice(0, DEFAULT_API_PAGE_SIZE) : null,
```

Comment:
Why does this need to be sliced? Doesn't the API return the correct number of results since we sent it a page param?

Reply:
mmh good catch, I think I don't need it anymore.